WooCommerce/WooCommerceTests/ViewRelated/Dashboard/CardReaderSettings/CardReaderSettingsKnownReadersStoredListTests.swift

```diff
@@ -0,0 +1,84 @@
+import XCTest
+import Combine
+@testable import WooCommerce
+
+/// Mock constants
+///
+private struct TestConstants {
+    static let mockReaderID = "CHB204909005931"
+}
+
+final class CardReaderSettingsKnownReadersStoredListTests: XCTestCase {
+
+    func test_subscribing_publishes_initial_value() {
```

Comment:
I added another test in 249d703fe since I don't think this one fully covered the initial value case. In `CardReaderSettingsKnownReadersStoredList`, we initialize the `knownReadersSubject` with an empty array, so even if we forgot to `loadReaders` or it failed for some reason, that subject would always have an empty initial value anyway.

In 249d703fe, I added another case to test whether the initial value was correct when there were previously known reader, which was passing correctly 😌